tensorboard/pip_package/requirements_dev.txt

```diff
@@ -17,6 +17,7 @@
 
 # For tests
 grpcio-testing==1.24.3
+pandas==1.0.3
```

Comment:
OK for now; we may want this to really be a Pip “extra”.

Reply:
Ack.